Documentation/guides/messages/xa4305.md

```diff
@@ -1,28 +1,24 @@
 ---
 title: Xamarin.Android warning XA4305
 description: XA4305 warning code
-ms.date: 10/30/2018
+ms.date: 02/11/2020
 ---
 # Xamarin.Android warning XA4305
 
-## Issue
+## Example messages
 
-The `CreateMultiDexMainDexClassList`, `CompileToDalvik` or `R8`
-MSBuild task encountered a *multidex.keep* file that was not found on
-disk. You can customize multidex settings for your Xamarin.Android
-application by adding files with the `MultiDexMainDexList` build item,
-which are combined into a final *multidex.keep* file.
+```
+Multidex is enabled, but `$(_AndroidMainDexListFile)` is empty.
+```
 
-To learn more about multidex and how it relates to Android
-development, see the [Android documentation][android].
+```
+Multidex is enabled, but the `$(_AndroidMainDexListFile)` 'obj\Release\90\multidex.keep' does not exist.
```

Comment:
I would suggest inserting "file" between `$(_AndroidMainDexListFile)` and `obj\...`, e.g.

```
Multidex is enabled, but the `$(_AndroidMainDexListFile)` file 'obj\Release\90\multidex.keep' does not exist.
```